WireAuthentication/Sources/WireAuthenticationUI/Views/Registration/PersonalAccountCreation/PersonalAccountCreationView.swift

```diff
@@ -0,0 +1,186 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireReusableUIComponents
+
+struct PersonalAccountCreationView: View {
+
+    @StateObject private var viewModel: PersonalAccountCreationViewModel
+
+    private typealias Strings = L10n.Localizable.CreatePersonalAccount
+    private typealias Labels = L10n.Accessibility.CreatePersonalAccount
+
+    package init(
+        factory: @autoclosure @escaping () -> PersonalAccountCreationFactory
+    ) {
+        self._viewModel = StateObject(wrappedValue: factory().viewModel)
+    }
+
+    package var body: some View {
+        ScrollView {
+            scrollViewContent
+                .navigationTitle(Strings.title)
+                .navigationBarTitleDisplayMode(.inline)
+                .setPreferredSize(navigationBarHidden: false)
+                .customBackButton()
+                .background(ColorTheme.Backgrounds.surface.color)
+        }
+        .navigationDestination(for: PersonalAccountCreationDestination.self) {
+            switch $0 {
+            case let .verifyEmail(email, password, name):
+                VerificationEmailCodeView(factory: viewModel.factory.verificationEmailCodeFactory(
+                    email: email,
+                    password: password,
+                    name: name
+                ))
+            }
+        }
```

Comment:
optional: extract the view into a private func, same for the sheet and the alert (if possible)